arbor/include/arbor/recipe.hpp

```diff
@@ -21,7 +21,7 @@ struct probe_info {
 };
 
 /* Recipe descriptions are cell-oriented: in order that the building
- * phase can be done distributedly and in order that the recipe
+ * phase can be done distributed, and in order that the recipe
```

Comment:
'distributedly', while awkward, is actually correct :)

Reply:
You are right. I fixed it by also removing the `done`, because `distributedly` is indeed both correct and funny sounding.